server/src/data-processors/helpers/verify-auth.js

```diff
@@ -0,0 +1,42 @@
+const requiredDependency = require('../../global-helpers/required-dependency');
+const throwError = require('../../global-helpers/throw-error');
+
+function makeVerifyAuth({ jwt = requiredDependency('jwt') } = {}) {
+  const jwtParamHasVerifyMethod = 'verify' in jwt;
+  if (!jwtParamHasVerifyMethod) {
+    requiredDependency('jwt');
+  } 
```

Comment:
What is the added benefit of checking for `verify()`? Doesn't JWT automatically come with `verify()`? Or is it a custom method you've made?

Reply:
In future, the `jwt` library used may not necessarily be the `jsonwebtoken` npm package. This gives anyone the ability to use whatever `jwt` dependency they want, as long as it works in the same way with a `verify` method.